### PR TITLE
update party_name and party_group

### PR DESCRIPTION
**Pull request by kalaisaranA**
updating party_name and party_group in Report Payment Period Based On Invoice Date.

### CODE REVIEW

**Comment by kalaisaranA** [comment]
By updating Party name and Party Group(customer/supplier group) gives detailed description on Party.kindly refer the snapshot
screenshot_2018-11-26 payment period based on invoice date

File: erpnext/accounts/report/payment_period_based_on_invoice_date/payment_period_based_on_invoice_date.py [hunk 1]
```diff
@@ -23,9 +23,22 @@ def execute(filters=None):
 			payment_amount = flt(d.debit) or -1 * flt(d.credit)
 		else:
 			payment_amount = flt(d.credit) or -1 * flt(d.debit)
+		if d.party_type == "Customer":
+			for i in frappe.db.sql("""
```
Comment (rmehta) [inline erpnext/accounts/report/payment_period_based_on_invoice_date/payment_period_based_on_invoice_date.py]: use the ORM

```
customer_name, customer_group = frappe.db.get_value('Customer', d.party, ['customer_name', 'customer_group'])
```

**Review by rmehta — CHANGES_REQUESTED** [review]
please use `frappe.db.get_value` instead of SQL

**Comment by sagarvora** [comment]
Reopen after changes.

**Pull request closed without merging**